bpf/lib/conntrack.h

```diff
@@ -884,7 +884,7 @@ static __always_inline int ct_create6(const void *map_main, const void *map_rela
 	if (dir == CT_INGRESS) {
```

Comment:
I think this should be wrapped also with `#ifdef CONNTRACK_ACCOUNTING` like in the lookup case to make it clear that accounting is enabled/disabled. Did you validate at runtime that by default it's disabled now?

Also, small nit: Please add a proper commit message so that it's not empty.

Otherwise lgtm then.